config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
> In case however there is a .env file

This PR doesn't add any support for parsing/using a `.env` file, the only reference to `.env` is my answer to ADmad's question.

> Another way to do it is to throw a HTTP 500 if a variable is set via $_SERVER and then is overwritten by .env. Hidden clashes are no fun (dangerous, error prone).

If you source a file on the cli, it's because you want to use the env vars defined in that file. It would be weird (not to mention obstructive and irritating) to define an env file causing the app to throw a 500.

Can you give a complete (and realistic) example/scenario of what you mean please - I suspect your objections stem from a misunderstanding.